src/vm/peimagelayout.cpp

```diff
@@ -172,7 +183,9 @@ void PEImageLayout::ApplyBaseRelocations()
                 dwOldProtection = 0;
             }
 
-            IMAGE_SECTION_HEADER *pSection = RvaToSection(rva);
+            USHORT fixup = fixupsCount ? VAL16(fixups[0]) : 0;
```

Comment:
It is hard to see whether the `fixupsCount==0` case works here. I think it would be better to handle it upfront and do nothing for it - add `if (fixupsCount ==0) continue;` right after the line that computes the `fixupsCount`.

It should not ever happen anyway.